tests/network/probes.go

```diff
@@ -80,8 +80,6 @@ var _ = SIGDescribe("[ref_id:1182]Probes", func() {
 
 				By("Specifying a VMI with a readiness probe")
 				vmi = createReadyCirrosVMIWithReadinessProbe(virtClient, readinessProbe)
-
```

Comment:
I'm ok with removing this assert. But if we want to be on the safe side, why not running an empty pod (with "while true sleep 1") and running the probe server on the pod just after the assert?

Reply:
@AlonaKaplan We also mentioned it on this thread https://github.com/kubevirt/kubevirt/pull/5317#issuecomment-813055162
If we want to be on the safe side we need to change the service inside the "backend pod" (netcat server)
so we could configure from the tests when it will actually work.
Then we can do the same flow as with the IPv4 test:
- Spin up VM with IPv6 readiness probe targeted to external service
- Assert that the VM&virt-launcher pod are not **not ready**
- Enable the service on the backend pod
- Asset VM&virt-launcher pod are **ready**
- 
I am planing to do this on a following PR, do you prefer to have it on this PR?

Reply:
There should be tests that check the probe fails when there is no service/ip, that should be enough.